assembl/static2/css/components/assembl-glyph.scss

```diff
@@ -116,3 +116,5 @@
 .assembl-icon-ellipsis:before { content: '\f141'; } /* '' */
 .assembl-icon-ellipsis-vert:before { content: '\f142'; } /* '' */
 .assembl-icon-sentiment-neutral:before { content: '\e86f'; } /* '' */
+.assembl-icon-double-page:before { content: '\e869'; } /* '' */
+.assembl-icon-timer:before { content: '\e86A'; } /* '' */
```

Comment:
Be careful, the font has been modified in #250 too

Reply:
Yes, I noticed it also while reviewing ^^ https://github.com/assembl/assembl/pull/250#discussion_r154043894
So, using Fontello instead of regular SVG files is not that good when developing several branches at the same time that make changes to icons :(